src/libsync/syncengine.cpp

```diff
@@ -214,6 +214,9 @@ bool SyncEngine::checkErrorBlacklisting( SyncFileItem &item )
         } else if( item._modtime != entry._lastTryModtime ) {
             qDebug() << item._file << " is blacklisted, but has changed mtime!";
             return false;
+        } else if( item._renameTarget != entry._renameTarget) {
+            qDebug() << item._file << " is blacklisted, byt renameTarget changed from" << entry._renameTarget;
```

Comment:
Could you also add a comment explaining this? Why is it not blacklisted then? for future...